pom.xml

```diff
@@ -20,36 +20,42 @@
     </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>org.utplsql</groupId>
-      <artifactId>java-api</artifactId>
-      <version>3.1.2</version>
-      <scope>compile</scope>
-        <exclusions>
-            <exclusion>
-                <groupId>com.oracle.jdbc</groupId>
-                <artifactId>ucp</artifactId>
-            </exclusion>
-        </exclusions>
-    </dependency>
-    <dependency>
-      <groupId>com.beust</groupId>
-      <artifactId>jcommander</artifactId>
-      <version>1.72</version>
-      <scope>compile</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.zaxxer</groupId>
-      <artifactId>HikariCP</artifactId>
-      <version>2.7.2</version>
-      <scope>compile</scope>
-    </dependency>
       <dependency>
-          <groupId>org.slf4j</groupId>
-          <artifactId>slf4j-nop</artifactId>
-          <version>1.7.25</version>
+          <groupId>org.utplsql</groupId>
+          <artifactId>java-api</artifactId>
+          <version>3.1.3-SNAPSHOT</version>
           <scope>compile</scope>
+          <exclusions>
+              <exclusion>
+                  <groupId>com.oracle.jdbc</groupId>
+                  <artifactId>ucp</artifactId>
+              </exclusion>
+          </exclusions>
       </dependency>
+      <dependency>
+          <groupId>com.beust</groupId>
+          <artifactId>jcommander</artifactId>
+          <version>1.72</version>
+          <scope>compile</scope>
+      </dependency>
+      <dependency>
+          <groupId>com.zaxxer</groupId>
+          <artifactId>HikariCP</artifactId>
+          <version>2.7.2</version>
+          <scope>compile</scope>
+      </dependency>
+      <dependency>
+          <groupId>javax.xml.bind</groupId>
+          <artifactId>jaxb-api</artifactId>
+          <version>2.3.0</version>
+      </dependency>
+      <dependency>
+          <groupId>ch.qos.logback</groupId>
```

Comment:
What's the reason behind choosing logback instead onf sl4j?

Reply:
slf4j is the logging framework (which is used), but it needs an implementation to run (which is logback because most lightweight with still the features needed)